src/xrCore/xrDebug.cpp

```diff
@@ -81,200 +80,200 @@ bool xrDebug::m_SymEngineInitialized = false;
-	{
-		Msg("[xrDebug::BuildStackTrace]InitializeSymbolEngine failed with error: %d", GetLastError());
-		return traceResult;
-	}
+    if (!InitializeSymbolEngine())
+    {
+        Msg("[xrDebug::BuildStackTrace]InitializeSymbolEngine failed with error: %d", GetLastError());
+        return traceResult;
+    }
 
-	traceResult.reserve(maxFramesCount);
+    traceResult.reserve(maxFramesCount);
 
 #if defined XR_X64
-	stackFrame.AddrPC.Mode = AddrModeFlat;
-	stackFrame.AddrPC.Offset = threadCtx->Rip;
-	stackFrame.AddrStack.Mode = AddrModeFlat;
-	stackFrame.AddrStack.Offset = threadCtx->Rsp;
-	stackFrame.AddrFrame.Mode = AddrModeFlat;
-	stackFrame.AddrFrame.Offset = threadCtx->Rbp;
+    stackFrame.AddrPC.Mode = AddrModeFlat;
+    stackFrame.AddrPC.Offset = threadCtx->Rip;
+    stackFrame.AddrStack.Mode = AddrModeFlat;
+    stackFrame.AddrStack.Offset = threadCtx->Rsp;
+    stackFrame.AddrFrame.Mode = AddrModeFlat;
+    stackFrame.AddrFrame.Offset = threadCtx->Rbp;
 #elif defined XR_X86	
-	stackFrame.AddrPC.Mode = AddrModeFlat;
-	stackFrame.AddrPC.Offset = threadCtx->Eip;
-	stackFrame.AddrStack.Mode = AddrModeFlat;
-	stackFrame.AddrStack.Offset = threadCtx->Esp;
-	stackFrame.AddrFrame.Mode = AddrModeFlat;
-	stackFrame.AddrFrame.Offset = threadCtx->Ebp;
+    stackFrame.AddrPC.Mode = AddrModeFlat;
+    stackFrame.AddrPC.Offset = threadCtx->Eip;
+    stackFrame.AddrStack.Mode = AddrModeFlat;
+    stackFrame.AddrStack.Offset = threadCtx->Esp;
+    stackFrame.AddrFrame.Mode = AddrModeFlat;
+    stackFrame.AddrFrame.Offset = threadCtx->Ebp;
 #else
 #	error CPU architecture is not supported.
 #endif
 
-	while (
-		GetNextStackFrameSring(&stackFrame, threadCtx, frameStr)
-		&&
-		traceResult.size() <= maxFramesCount
-		)
-	{
-		traceResult.push_back(frameStr);
-	}
+    while (
+        GetNextStackFrameString(&stackFrame, threadCtx, frameStr)
+        &&
+        traceResult.size() <= maxFramesCount
+        )
+    {
+        traceResult.push_back(frameStr);
+    }
 
-	DeinitializeSymbolEngine();
+    DeinitializeSymbolEngine();
 
-	return traceResult;
+    return traceResult;
 }
 
 SStringVec xrDebug::BuildStackTrace(u16 maxFramesCount)
-{	
-	CONTEXT currentThreadCtx = { 0 };	
+{
+    CONTEXT currentThreadCtx = { 0 };
 
-	RtlCaptureContext(&currentThreadCtx);	/// GetThreadContext cann't be used on the current thread 
-	currentThreadCtx.ContextFlags = CONTEXT_FULL;
+    RtlCaptureContext(&currentThreadCtx);	/// GetThreadContext cann't be used on the current thread 
+    currentThreadCtx.ContextFlags = CONTEXT_FULL;
 
-	return BuildStackTrace(&currentThreadCtx, maxFramesCount);
+    return BuildStackTrace(&currentThreadCtx, maxFramesCount);
 }
 
 void xrDebug::LogStackTrace(const char* header)
 {
-	SStringVec stackTrace = BuildStackTrace();
-	Msg("%s", header);
-	for each (const auto& frame in stackTrace)
-	{
-		Msg("%s", frame.c_str());
-	}
+    SStringVec stackTrace = BuildStackTrace();
+    Msg("%s", header);
+    for(const auto& frame : stackTrace)
```

Comment:
Put one space after keywords;
Put pointer and reference next to the identifier;
https://github.com/OpenXRay/xray-16/wiki/Cpp-Coding-Conventions